k6/docker-entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -e
+
+# Check for the test case file via either a passed-in env path or the default
+# In either case, the file should be mounted as a volume (from configmap, etc)
+TEST_CASE_FILE=${TEST_CASE_FILE:-"/scripts/load_script.js"}
+TEST_CASE_OUTPUT=${TEST_CASE_OUTPUT:-"prometheus.txt"}
+
+if ! [ -f "${TEST_CASE_FILE}" ]; then
+        echo "A test case JS file must be provided." 1>&2
+        exit 1
+fi
+
+# Check that we have a Prometheus Pushgateway defined
+if [ -z "${PUSHGATEWAY_URL}" ]; then
+        echo "No Pushgateway URL" 1>&2
+				exit 1
```

Comment:
Make sure to use tabs for indentation in shell scripts.

Reply:
Changes made as specified.